### ISSUE TITLE
Crash when searching a video

### ISSUE THREAD

**Issue by egdel** [Gericom/YoutubeDS#1]
The homebrew hangs instantly when i try to search a video.

**Comment by Gericom** [comment]
Yeah, the https to http proxy is not up anymore. I'm sorry. If you host it somewhere you could probably build a new version that uses a different proxy

**Comment by ayunami2000** [comment]
im hosting an http proxy http://enable-cors.glitch.me just append the url to proxy to the end of the URL (e.g. http://enable-cors.glitch.me/https://www.youtube.com) to use now BUILD AND RELEASE YOUTUBEDS PLS

**Comment by Gericom** [comment]
Ah, that's nice! I'll see if I can get it working again when I have time

**Comment by ayunami2000** [comment]
i also have a youtube search and download api at http://tubedown.glitch.me but the downloader doesnt work with all videos so you would have to download them somehow. example: [https://tubedown.glitch.me/~youtube ds preview](https://tubedown.glitch.me/~youtube%20ds%20preview). it returns json of what searching on mobile youtube would give you. nobody currently uses the api so this would be a good use

**Comment by Gericom** [comment]
That's great!

**Comment by s12227054** [comment]
> That's great!

Did you get it working?

**Comment by Natemo2625** [comment]
Is https://tubedown.glitch.me broken @ayunami2000 and did you delete https://enable-cors.glitch.me

**Comment by ayunami2000** [comment]
No they both work, tube down isn't 100% reliable though and I suggest using
an invidious instance. Maybe allow the user to select their preferred
invidious host?

**Comment by ayunami2000** [comment]
https://redirect.invidious.io/ for invidious instances to use. API is
http://[invidious host]/latest_version?id=[video id]&itag=[itag]
Sample itag is 18 for a low quality video

**Comment by ayunami2000** [comment]
Also if you want to use it here is the api for instances
https://api.invidious.io/instances.json?sort_by=type,health

**Comment by ten-faced-carrot** [comment]
For anyone still here, I've done *a bit* of digging. It seems like, for... some reason? The HTTP Request isn't even going out - I've tinkered around with my pihole and looked at the DNS Requests... Yeah, it's not sending out ANYTHING - even though I have verified that it is, in fact, using that DNS (The Connection Test showed up successfully actually!)

I'm gonna try tinkering around a bit more

**Comment by Natemo2625** [comment]
Looking forward to seeing your progress @ten-faced-carrot

**Comment by asticassiasuletta** [comment]
holy shit, still here 4 years later? that's incredible

**Comment by coolgamer12378** [comment]
this still work??

**Comment by XRTgaming** [comment]
bro none of these work in australia someone do it for me please
